docs/source/install_devel.rst

```diff
@@ -80,4 +80,4 @@ The development version of cocotb can be installed by running
 
 After installation, you should be able to execute ``cocotb-config``.
 If it is not found, you need to append its location to the ``PATH`` environment variable.
-This may happen when you use the ``--user`` option to ``pip``, in which case the location is documented :ref:`here<python:inst-alt-install-user>`.
+This may happen when you use the ``--user`` option to ``pip``.
```

Comment:
This section of the documentation no longer exists, it's been totally rewritten and there isn't an equivalent section to link to.